xml/article_sap_trento.xml

```diff
@@ -40,36 +40,42 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
   <info>
     <abstract>
       <para>
+       <remark>toms 2022-04-27: change 1</remark>
        &trentopremium; is an open cloud-native web console that aims to help
-       &sap; administrators to improve monitoring and checking of &sap;
-       Applications.
+       &sap; Basis consultants and administrators to monitor the entire OS stack of their
+       &sap; environments, including HA features.
       </para>
     </abstract>
     <xi:include href="common_copyright_gfdl.xml" />
     </info>
 
   <important>
     <title>Trento is an evolving product</title>
-    <para> The product is under active development, and it currently
-      supports only selected customers with the purpose of collecting
-            their feedback. If you'd like to join the program, contact us at
-                <email>trento-project@suse.com</email>. </para>
+    <para>
+      <remark>toms 2022-04-27: change 2</remark>
+      The product continues to be under active development but is available for any &suse;
+      customer with a registered &productname; 15 (SP1 or higher). Issues should be reported
+      through <link xlink:href="https://bugzilla.suse.com"/> under product
+      <productname>SUSE Linux Enterprise Server for SAP Applications 15 (SP1 or higher)</productname>
```

Comment:
@abravosuse , @tomschr 
We cannot reference bugzilla.suse.com as those products are internal, let's talk about the "usual support way". Toms, it's the default way for contacting support, what do we have in other manuals? Let's use that wording...

Reply:
@ajaeger The "usual support way" can be to link to www.suse.com/support (which we do already in some chapters). Would the following change sufficient?

```
The product continues to be under active development, but is available for any &suse;
customer with a registered &productname; 15 (SP1 or higher).
Contact https://www.suse.com/support for further assistence.
```

Reply:
Works for me, thanks!